flow/project.py

```diff
@@ -1603,16 +1972,35 @@ def _get_operations_status(self, job, cached_status):
         starting_dict = functools.partial(dict, scheduler_status=JobStatus.unknown)
         status_dict = defaultdict(starting_dict)
         for group in self._groups.values():
-            completed = group.complete(job)
-            eligible = False if completed else group.eligible(job)
-            scheduler_status = cached_status.get(group._generate_id(job),
+            jobs_list = group._get_filtered_jobs(self)
```

Comment:
Shouldn't this job operate on a `job_list` parameter now, given the signature with a `job` parameter.